SolarTherm/Materials/Sodium_Table.mo

```diff
@@ -0,0 +1,57 @@
+within SolarTherm.Materials;
```

Comment:
please clarify how this file relates to the earlier sodium 'motab' file. are both required? is one of them redundant?